cmake fails on Windows

```
CMake Error in CMakeLists.txt:
  Property "INTERFACE_COMPILE_OPTIONS" of target "fmt" contains a generator
  expression.  This is not allowed.


CMake Error in CMakeLists.txt:
  Property "INTERFACE_COMPILE_OPTIONS" of target "fmt-header-only" contains a
  generator expression.  This is not allowed.


CMake Generate step failed.  Build files cannot be regenerated correctly.
```

likely because externpro is using
```cmake
install(PACKAGE_INFO
```
to install a [Common Package Specification](https://cps-org.github.io/cps/) ("CPS") file exporting targets for dependent projects

the blog announcement "Common Package Specification is Out the Gate" https://www.kitware.com/common-package-specification-is-out-the-gate/ mentions in the "CMake / CPS Roadmap" section some of the "rough edges", "remaining issues", and "known limitations"
>**Generator Expression Support.** While there are no immediate plans to support generator expressions as such in CPS, this means that CMake exports that depend on generator expressions cannot be exported as CPS. Some of these limitations, however, are artificial, as a number of “generator expressions” are not actually dynamic (that is, they could be evaluated at configure-time) or vary only on axes that CPS otherwise models (namely, configuration or compile language).

and this appears to be the issue, as fmt has the following at https://github.com/externpro/fmt/blob/12.1.0/CMakeLists.txt#L367-L373
```cmake
if (NOT MSVC)
  # Unicode is always supported on compilers other than MSVC.
elseif (FMT_UNICODE)
  # Unicode support requires compiling with /utf-8.
  target_compile_options(fmt PUBLIC $<$<AND:$<COMPILE_LANGUAGE:CXX>,$<CXX_COMPILER_ID:MSVC>>:/utf-8>)
  target_compile_options(fmt-header-only INTERFACE $<$<AND:$<COMPILE_LANGUAGE:CXX>,$<CXX_COMPILER_ID:MSVC>>:/utf-8>)
else ()
  target_compile_definitions(fmt PUBLIC FMT_UNICODE=0)
endif ()
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -369,8 +369,8 @@ if (NOT MSVC)
   # Unicode is always supported on compilers other than MSVC.
 elseif (FMT_UNICODE)
   # Unicode support requires compiling with /utf-8.
-  target_compile_options(fmt PUBLIC $<$<AND:$<COMPILE_LANGUAGE:CXX>,$<CXX_COMPILER_ID:MSVC>>:/utf-8>)
-  target_compile_options(fmt-header-only INTERFACE $<$<AND:$<COMPILE_LANGUAGE:CXX>,$<CXX_COMPILER_ID:MSVC>>:/utf-8>)
+  target_compile_options(fmt PUBLIC /utf-8)
+  target_compile_options(fmt-header-only INTERFACE /utf-8)
 else ()
   target_compile_definitions(fmt PUBLIC FMT_UNICODE=0)
 endif ()

